tasks/nix.sh

```diff
@@ -30,7 +30,12 @@ else
   timeout_min=$(($timeout/60))
   timeout_sec=$(($timeout%60))
   nohup bash -c "sleep $timeout_sec; shutdown -r +$timeout_min $message" </dev/null >/dev/null 2>&1 &
+  disown
```

Comment:
Not sure what scenario would make this happen, but based on https://unix.stackexchange.com/a/148698 I don't believe it hurts.